maps/offmap_vr/om_ships/bearcat.dm

```diff
@@ -0,0 +1,200 @@
+// Compile in the map for CI testing if we're testing compileability of all the maps
+#if MAP_TEST
+#include "bearcat.dmm"
+#endif
+
+// -- Datums -- //
+/datum/map_template/om_ships/bearcat
+	name = "OM Ship - Bearcat (New Z)"
+	desc = "An old salvage ship, abandoned but seemingly intact."
+	mappath = 'bearcat.dmm'
+
+// The 'shuttle'
+/datum/shuttle/autodock/overmap/bearcat
+	name = "Bearcat"
+	current_location = "omship_spawn_bearcat"
+	docking_controller_tag = "bearcat_starboard" //technically we have two ports, but the starboard is used as the default as it allows us to hook up to most sites
+	shuttle_area = list(	/area/ship/scrap,
+				/area/shuttle/bearcat/cargo,
+				/area/shuttle/bearcat/command,
+				/area/shuttle/bearcat/command/captain,
+				/area/shuttle/bearcat/comms,
+				/area/shuttle/bearcat/crew,
```

Comment:
Try repathing the areas in the shuttle_area list so that you're not including anything that also has a subtype in the list, that has caused issues for me in the past. So, `bearcat/crew/whatevers`, but no `bearcat/crew` in the same shuttle.

Reply:
Yeah, that seems to have done it. I'd tried removing a couple before but I wasn't very thorough with it. Seems to be behaving now after some find+replace operations.